test/nodetool/README.md

```diff
@@ -0,0 +1,23 @@
+# Front-end tests for nodetool
+
+The tests in this directory excercise the nodetool client itself, mocking the API backend.
+This allows for testing all combinations of all supported options, and still keeping the tests quick.
+
+The tests can be run against both the scylla-native nodetool (default), or the inherited, C\*-based nodetool.
+
+Run all tests against the scylla-native nodetool:
+```
+pytest --nodetool=scylla .
+```
+
+You can specify the path to the scylla binary with the `--nodetool-path` option. By default the tests will pick up the ScyllaDB executable, that is appropriate for the `--mode` option (defaults to `dev`).
+
+Run all tests against the C* nodetool:
+```
+pytest --nodetool=cassandra .
+```
+
+Again, you can specify the path to the nodetool binary with `--nodetool-path` option. By default, `scylladb.git/tools/java/bin/nodetool` will be used.
+You can specify the path to JMX with `--jmx-path` option. By default, `scylladb.git/tools/jmx/scripts/scylla-jmx` will be used.
```

Comment:
Can you please explain in more detail what JMX is used for? I assume it's only needed for the "--nodetool=cassandra" case, and not needed for the Scylla-native nodetool?

Reply:
Yes, therefore I only mention `JMX` here at the section about `--nodetool=cassandra`.

Reply:
Ok, I didn't understand it was inside some specific section.

Reply:
Added further clarifications in v2.